### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by yaz-saleh**
Cleaned up public module and customized it for Validere's purposes/use-case.
Might be easiest to review by viewing the branch [directly](https://github.com/ValidereInc/grafana-fargate/tree/cleanup) rather than going through the changes.

### CODE REVIEW

File: terraform/rds.tf [hunk 1]
```diff
@@ -33,11 +33,15 @@ resource "aws_db_subnet_group" "grafana" {
   }
 }
 
+data "aws_secretsmanager_secret_version" "grafana_db_backend" {
+  secret_id = "grafana-backend-db-creds"
+}
+
```
Comment (RobBlumberg) [inline terraform/rds.tf]: thinking we should be able to have different passwords for the different downstream envs, so like 
```suggestion
data "aws_secretsmanager_secret_version" "grafana_db_backend" {
  secret_id = format("%s-grafana-backend-db-creds", var.env)
}
```

File: terraform/grafana.tf [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
-    log_group_region = var.region
-    log_group_name   = aws_cloudwatch_log_group.grafana.name
+locals {
+  grafana_config = {
+    GF_SERVER_DOMAIN     = var.dns_name
+    GF_DATABASE_USER     = var.grafana_db_username
+    GF_DATABASE_TYPE     = "mysql"
+    GF_DATABASE_HOST     = "${aws_rds_cluster.grafana.endpoint}:3306"
+    GF_LOG_LEVEL         = var.grafana_log_level
+    GF_DATABASE_PASSWORD = jsondecode(data.aws_secretsmanager_secret_version.grafana_db_backend.secret_string).password
```
Comment (RobBlumberg) [inline terraform/grafana.tf]: By the looks of it [here](https://scicoding.com/aws-cognito-authorication/), to add cognito as the authorization server, we would just need to add the relevant env vars for OAuth?

**Review by RobBlumberg — APPROVED** [review]
LGTM just a few comments. Approving so you can merge once addressed.

Also:
- Wondering if you think [these autoscaling policies](https://github.com/ValidereInc/grafana-fargate/blob/93a042d07971797b8b3456ab59cf5ef09e2b0074/terraform/cw.tf) that trigger from different cw alarms are desired for our case? Do you think its something that is needed? Maybe we should at least have a tf var to toggle them on/off (and probably start with them off, at least for like the dev / integration envs)

**Note:** Reviewed along with https://github.com/ValidereInc/va_monitoring/pull/4/ and https://github.com/ValidereInc/analytics_serverless_apps/pull/266

File: terraform/rds.tf (hunk 1 above)
Comment (yaz-saleh) [reply terraform/rds.tf]: True, addressed - I guess in theory shouldn't matter now that we have (or will have) separate accounts for each env but good to stay consistent.

File: terraform/grafana.tf (hunk 2 above)
Comment (yaz-saleh) [reply terraform/grafana.tf]: Yep I think so! I may give it a try/squeeze it in if I have some time.

**Comment by yaz-saleh** [comment]
> LGTM just a few comments. Approving so you can merge once addressed.
> 
> Also:
> 
> * Wondering if you think [these autoscaling policies](https://github.com/ValidereInc/grafana-fargate/blob/93a042d07971797b8b3456ab59cf5ef09e2b0074/terraform/cw.tf) that trigger from different cw alarms are desired for our case? Do you think its something that is needed? Maybe we should at least have a tf var to toggle them on/off (and probably start with them off, at least for like the dev / integration envs)
> 
> **Note:** Reviewed along with [ValidereInc/va_monitoring#4](https://github.com/ValidereInc/va_monitoring/pull/4) and [ValidereInc/analytics_serverless_apps#266](https://github.com/ValidereInc/analytics_serverless_apps/pull/266)

I think they are fine to keep since in theory the int/dev envs would not scale needlessly if not triggered, which is likely going to be the case on those envrionments.. so in essence it is the equivalent of having them off, no? Let me know if you had other thoughts

**Pull request merged**